analysis/deep2_r23_o32/stackboth_mastergrid.py

```diff
@@ -82,13 +82,13 @@ def master_stacking(fitspath, fitspath_ini, dataset, grid_data_file,
         image2DM = np.ma.masked_array(image2DM, maskM[det3])
         log.debug("Mask[det3]", maskM[det3])
 
-    outfile = join(fitspath, filename_dict['comp_spec'])
-    if not exists(outfile):
-        stack_2d = np.zeros((len(R23_minimum) * len(O32_minimum), len(wave)),
+    stack_2d_file = join(fitspath, filename_dict['comp_spec'])
+    if not exists(stack_2d_file):
+        stack_2d_tab = np.zeros((len(R23_minimum) * len(O32_minimum), len(wave)),
                             dtype=np.float64)
```

Comment:
Fix indentation here.

Reply:
please check to make sure I fixed the problem you spotted